### PR TITLE
Create aws.txt

### PR DESCRIPTION
**Pull request by sf-matt**

### CODE REVIEW

File: aws.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+AWS_ACCESS_KEY_ID:  ********************
+AWS_SECRET_ACCESS_KEY:  0+rfumaQJh8p+ViUGaBTeCgimS2ukgGnKrLQI9RR
```
Comment (gnair) [inline aws.txt:2]: :exclamation:**Cycode**: Secret of type: 'Aws Secret Access Key' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

Comment (sf-matt) [reply aws.txt:2]: #cycode_secret_revoked

Comment (gnair) [reply aws.txt:2]: :-1: This user is not permitted to perform actions.

Comment (sf-matt) [reply aws.txt:2]: #cycode_secret_revoked

Comment (sf-matt) [reply aws.txt:2]: :-1: Cycode cannot perform further actions in this thread.

**Pull request closed without merging**